tools/smithy-rs-sync/src/main.rs

```diff
@@ -245,9 +261,31 @@ fn set_last_synced_commit(repo_path: &Path, oid: &Oid) -> Result<()> {
         .with_context(|| format!("Couldn't write commit hash to '{}'", path.display()))
 }
 
+/// Place the examples from aws-doc-sdk-examples into the correct place in smithy-rs
+/// to be included with the generated SDK.
+fn setup_examples(sdk_examples_path: &Path, smithy_rs_path: &Path) -> Result<()> {
+    let from = sdk_examples_path.canonicalize().context(here!())?;
+    let from = from.join("rust_dev_preview");
+    let from = from.as_os_str().to_string_lossy();
+
+    eprintln!("\tcleaning examples...");
+    let _ = run(&["rm", "-rf", "aws/sdk/examples"], smithy_rs_path).context(here!())?;
```

Comment:
We should probably use `std::fs` when it's easy to do so:
```suggestion
    let _ = std::fs::remove_dir_all(smithy_rs_path.join("aws/sdk/examples")?;
```